specification/communication/data-plane/Rooms/preview/2022-02-01-preview/communicationservicesrooms.json

```diff
@@ -495,69 +371,65 @@
     "RoomParticipant": {
       "description": "A participant of the room.",
       "required": [
-        "communicationIdentifier"
+        "rawId",
+        "role"
       ],
       "type": "object",
       "properties": {
-        "communicationIdentifier": {
-          "$ref": "#/definitions/CommunicationIdentifierModel"
+        "rawId": {
+          "description": "Raw id of the identifier.",
+          "type": "string"
         },
         "role": {
-          "$ref": "#/definitions/RoleType"
+          "$ref": "#/definitions/Role"
         }
       }
     },
-    "CommunicationIdentifierModel": {
-      "description": "Identifies a participant in Azure Communication services. A participant is, for example, an Azure communication user. This model must be interpreted as a union: Apart from rawId, at most one further property may be set.",
+    "InvitedRoomParticipant": {
+      "description": "A participant invited to the room.",
+      "required": [
+        "rawId"
+      ],
       "type": "object",
       "properties": {
         "rawId": {
-          "description": "Raw id of the identifier. Optional in requests, required in responses.",
+          "description": "Raw id of the identifier.",
```

Comment:
```suggestion
          "description": "Raw ID representation of the communication identifier.",
```

Reply:
Similar to above comment, keep the original description

Reply:
Updated the description and added doc link